<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFVideoPlyerClass</class>
 <widget class="QMainWindow" name="FFVideoPlyerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFVideoPlyer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="7">
     <widget class="QLabel" name="label_cache">
      <property name="text">
       <string>缓存帧数：0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="label_FPS">
      <property name="text">
       <string>FPS：0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="7">
     <widget class="QLineEdit" name="lineEditURL">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioButton_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="radioButton_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_1">
        <property name="text">
         <string>1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="7">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="btn_openURL">
        <property name="text">
         <string>打开URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_openVideo">
        <property name="text">
         <string>打开视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_openImg">
        <property name="text">
         <string>打开图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="5" colspan="7">
     <widget class="VideoViewWidget" name="openGLWidget"/>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>模拟键盘输出的字符:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="checkDeal">
        <property name="text">
         <string>开启识别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>处理帧率:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxFPS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btn_REC">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>REC</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="btn_UP">
      <property name="text">
       <string>上移</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="btn_Down">
      <property name="text">
       <string>下移</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>调整选框位置：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QCheckBox" name="checkShow">
      <property name="text">
       <string>显示</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="btn_Left">
      <property name="text">
       <string>左移</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="btn_Right">
      <property name="text">
       <string>右移</string>
      </property>
     </widget>
    </item>
   </layout>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoViewWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videoviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
